contrib/packaging/docker/Dockerfile.runtime

```diff
@@ -8,7 +8,7 @@ apt-get upgrade -y && \
 # Prepackaged Cilium runtime dependencies
 #
 apt-get install -y --no-install-recommends \
-    gpg gpg-agent libelf-dev libmnl-dev libc6-dev-i386 iptables libgcc-5-dev \
```

Comment:
Just tried this locally to after you reported it. It seems this breaks the `bpftool` build:

```
In file included from skeleton/profiler.bpf.c:4:
In file included from /usr/include/linux/bpf.h:11:
/usr/include/linux/types.h:5:10: fatal error: 'asm/types.h' file not found
#include <asm/types.h>
         ^~~~~~~~~~~~~
1 error generated.
```

I thought that installing `libc-linux-dev:amd64` (which provides said header) would help but for some reason it doesn't.

Reply:
The problem seems to be is that the `/usr/include/asm` symlink is only created once the headers of another arch (in this case for `i386` are installed). For cilium, PR #10204 thus added `-I/usr/include/$(shell uname -m)-linux-gnu` to `$CFLAGS`. But it seems `bpftool` doesn't do that.

Reply:
True, we can move it to the `runtime-build` section though since our BPF code generation does not depend on it anymore. And that said, there's move *-dev stuff we could probably move to there, will check.

Reply:
> The problem seems to be is that the `/usr/include/asm` symlink is only created once the headers of another arch (in this case for `i386` are installed). For cilium, PR #10204 thus added `-I/usr/include/$(shell uname -m)-linux-gnu` to `$CFLAGS`. But it seems `bpftool` doesn't do that.

Could you craft a patch for upstream bpftool on that?

Reply:
Sure, sent https://lore.kernel.org/bpf/20200311123421.3634-1-tklauser@distanz.ch/T/#u

Just noticed that there's a typo in the title: `s/iprofile/profile/`